app/src/components/market/sections/market_create/market_wizard_creator.tsx

```diff
@@ -38,15 +38,24 @@ export const MarketWizardCreator = (props: Props) => {
     collateral: defaultCollateral,
     funding: new BigNumber('0'),
     loadedQuestionId: null,
-    outcomes: [],
+    outcomes: [
+      {
+        name: '',
+        probability: 50,
+      },
+      {
+        name: '',
+        probability: 50,
+      },
+    ],
     question: '',
     resolution: null,
     spread: MARKET_FEE,
   }
 
   const [currentStep, setCurrentStep] = useState(1)
   const [marketData, setMarketdata] = useState<MarketData>(marketDataDefault)
-  const [first, setFirst] = useState<number>(8)
+  const [first, setFirst] = useState<number>(19)
```

Comment:
I don't think any action needs to be taken, but as a sort of mental note, once the subgraph is updated to ignore uppercase categories, we will need to switch this back so we're not overflowing with categories to choose from.